dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -330,6 +333,53 @@ def kurtosis(self):
                     self.match_count,
                     self._biased_kurtosis)
 
+    @staticmethod
+    def _perform_t_test(mean1, var1, n1,
+                        mean2, var2, n2):
+        results = {}
+        s_delta = var1/n1 + var2/n2
+        t = (mean1 - mean2) / np.sqrt(s_delta)
+        conservative_df = min(n1, n2) - 1
+        welch_df = s_delta ** 2 / ((var1 / n1) ** 2 / (n1 - 1) + (var2 / n2) ** 2 / (n2 - 1))
+        results.update({
+            "t-statistic": t,
+            "conservative_df": conservative_df,
```

Comment:
if instead results were:
```python
{
    {
        "t-statistic": ,
        "conservative": {
            "df": <VALUE>,
            "p-value": <NONE_OR_VALUE>
        },
        "welch": {
            "df": <VALUE>,
            "p-value": <NONE_OR_VALUE>
        }
        
}
```
then we wouldn't have multiple sets of info and it would all be consistent whether it was computed or not.
`None` being the value for not computed.

Reply:
Agreed, this looks a lot cleaner